crates/sui-indexer/src/lib.rs

```diff
@@ -66,3 +73,24 @@ pub fn get_pg_pool_connection(
         ))
     })
 }
+
+pub async fn build_json_rpc_server(
+    prometheus_registry: &Registry,
+) -> Result<ServerHandle, IndexerError> {
+    let builder = ExtendedJsonRpcServerBuilder::new(FAKE_PKG_VERSION, prometheus_registry)
```

Comment:
why do we need the new crate and `ExtendedJsonRpcServerBuilder`? 

we should be able to re-use `JsonRpcServerBuilder` here directly?

Reply:
you are right, I was originally thinking that we will have similar directory & file names for APIs, so it is cleaner to separate them, but they do not have to be a different crate, I can add a new sub-dir for that.